ekncontent/overrides/EosKnowledgeContent.js

```diff
@@ -54,14 +51,6 @@ function add_custom_model_constructors (model) {
 
 function _init() {
     let Eknc = this;
-
-    define_property(Eknc.ContentObjectModel, 'discovery-feed-content', {
-        get: function () {
-            let content = this.get_discovery_feed_content();
-            return content ? content.deep_unpack() : [];
-        }
-    });
```

Comment:
It's more a case of unpacking GVariants into JS types; the GVariant type is pretty nasty from a non-C perspective.

Reply:
I removed the custom getter and setter because they never worked in the first place; `this.get_discovery_feed_content()` didn't exist. However, I'm now reworking this to add that accessor function, so I'll keep the custom getter and setter.